systers_portal/blog/models.py

```diff
@@ -64,3 +66,17 @@ def get_absolute_url(self):
         return reverse('view_community_resource',
                        kwargs={'slug': self.community.slug,
                                'resource_slug': self.slug})
+
+
+class UserPins(models.Model):
+    user = models.OneToOneField(SystersUser, on_delete=models.CASCADE)
+    pins = models.ManyToManyField(Resource, blank=True, related_name="pins", verbose_name="pins")
+
+    def __str__(self):
+        return "Pins for {0}".format(self.user)
+
+    def add_pin(self, resource):
+        self.pins.add(resource)
```

Comment:
Why not `self.save()` here?

Reply:
Are you talking about adding self.save() after? Because pins is a manyToMany field

Reply:
Because he has saved the pin object when he creates it the first time. That should work

Reply:
Oh I have never worked with them, so that makes most of my comments irrelevant

Apologies @ritwickraj78, feel free to ignore most of these comments